addons/website_quote/models/sale_order.py

```diff
@@ -190,15 +190,15 @@ def get_access_action(self, access_uid=None):
 
     def get_mail_url(self):
         self.ensure_one()
-        if self.state not in ['sale', 'done']:
+        if self.template_id and self.state not in ['sale', 'done']:
```

Comment:
Hi,
Why that change ?

Reply:
We can skip this option, as the functionality has been changed in latest odoo v11 with https://github.com/odoo/odoo/pull/21971/commits/b7a064af9f2e1f9e9a4d7c28b7bcbb8af7842f4e.